go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/aquasecurity/table v1.8.0
 	github.com/aquasecurity/testdocker v0.0.0-20240730042311-4642e94c7fc8
 	github.com/aquasecurity/tml v0.6.1
-	github.com/aquasecurity/trivy-checks v0.13.1-0.20240830035934-7761a83288cd
+	github.com/aquasecurity/trivy-checks v0.13.1-0.20240830230553-53ddbbade784
```

Comment:
I'll change this to the next release tag, once we release trivy-checks to be consistent.